### PR TITLE
Vindictively disable all intermittent tests that failed #9787

### PR DESCRIPTION
**Pull request by nox**
This disables the tests mentioned in #8120, #9014, #9033, #9092, #9148, #9205, #9772 and #9803.

[Review on Reviewable](https://reviewable.io/reviews/servo/servo/9804)

### CODE REVIEW

**Comment by nox** [comment]
@bors-servo r=KiChjang

I failed the disabling of #9803.

File: tests/wpt/metadata/websockets/cookies/006.html.ini [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[005.html]
```
Comment (KiChjang) [inline tests/wpt/metadata/websockets/cookies/006.html.ini]: Why is this 005 when the filename is 006?

**Comment by nox** [comment]
I fixed that thing.

@bors-servo r=KiChjang

**Pull request merged**